lib/ret_web/controllers/api/v1/app_config_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule RetWeb.Api.V1.AppConfigController do
+  use RetWeb, :controller
+  alias Ret.{Repo, AppConfig}
+
+  def create(conn, %{"app_config" => app_config_params}) do
```

Comment:
i'm not familiar with the APIs yet, but I'm guessing we should make this basically the same API as `ita` (or vice versa)

Reply:
(and/or we may want to have this API proxy to ita for configs that are managed there)